test/sql/optional_columns.test

```diff
@@ -25,7 +25,7 @@ EDGE TABLES (
 query IIIIIIIIIII
 -FROM GRAPH_TABLE (snb MATCH (p:Person)) limit 1;
 ----
-2010-01-03 23:10:31.499+00	14	Hossein	Forouhar	male	1984-03-11	77.245.239.11	Firefox	1166	fa;ku;en	Hossein14@hotmail.com
+1166	1984-03-11	Firefox	2010-01-03 23:10:31.499+00	Hossein14@hotmail.com	Hossein	male	14	Forouhar	77.245.239.11	fa;ku;en
```

Comment:
NOTICE: This is a behavior change.
- For star expression, duckdb returns columns in the order of they are declared in the `CREATE` statement;
- While I'm directly using `columns` in the property graph table: https://github.com/cwida/duckdb-pgq/blob/c380346433db588352999594227f2f8d23be2f49/src/include/duckdb/parser/property_graph_table.hpp#L33
- I think it's ok since they're actually two different SQL statements, so don't need to seek for completely compatibility here; but if it's required, I need to check how property graph table is constructed and why is the ordering difference.